src/Components/addItem.js

```diff
@@ -1,40 +1,94 @@
 import React from 'react';
+import '../App.css';
 import { fb } from '../lib/firebase';
 
 class AddItem extends React.Component {
   constructor() {
     super();
     this.state = {
       itemName: '',
+      purchaseFrequency: null,
+      lastPurchasedDate: null,
     };
   }
   updateInput = e => {
     this.setState({
       [e.target.name]: e.target.value,
     });
   };
+
+  updateFrequency = e => {
+    this.setState({
+      purchaseFrequency: e.target.value,
+    });
+  };
+  /* user is placeholder for user token when functionality is completed */
   addItem = e => {
     e.preventDefault();
     const db = fb.firestore();
-    db.collection('items').add({
+    db.collection('user').add({
       itemName: this.state.itemName,
+      purchaseFrequency: this.state.purchaseFrequency,
+      lastPurchasedDate: this.state.lastPurchasedDate,
     });
+
     this.setState({
       itemName: '',
+      purchaseFrequency: null,
+      lastPurchasedDate: null,
     });
   };
   render() {
     return (
-      <form onSubmit={this.addItem}>
-        <input
-          type="text"
-          name="itemName"
-          onChange={this.updateInput}
-          value={this.state.itemName}
-        />
-        <button type="submit">Submit</button>
-      </form>
+      <div className="add-item-form">
+        <form onSubmit={this.addItem}>
+          <label id="item-text">
+            Name of Item
+            <input
+              type="text"
+              name="itemName"
+              onChange={this.updateInput}
+              value={this.state.itemName}
+            />
+          </label>
+          <br></br>
+          <label>How soon do you expect to buy this again?</label>
+          <br></br>
+          <div className="form-button">
+            <label>
+              <input
```

Comment:
I would _love_ to see a style applied to these `<input type="radio">` elements so we can easily see which one we've clicked before we submit.

![](https://cdn.zappy.app/3134519c4c78a5d819e87e1f3d2de3ba.gif)

Because you're already defining `checked` on line 62, I think you could add some CSS for `input:checked`, or if you add a className like `className="frequencyInput"` you could do `.frequencyInput:checked`. Just changing the [`background-color`](https://developer.mozilla.org/en-US/docs/Web/CSS/background-color) would be enough here!

Check out [the documentation on MDN](https://developer.mozilla.org/en-US/docs/Web/CSS/:checked) to learn more about the `:checked:` selector.